### PR TITLE
Dev > Master (2.5 release)

### PR DESCRIPTION
**Pull request by crfroehlich**

### CODE REVIEW

File: assets/configuration/core_configuration.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (davehensleyns8) [inline assets/configuration/core_configuration.json:2]: Do we want this to start matching the actual SDK version?

File: assets/configuration/core_configuration.json [hunk 2]
```diff
@@ -15,22 +15,25 @@
       "enabled": true,
       "log_level": "INFO"
     },
-    "record_all_http_calls": true
+    "record_all_http_calls": false
```
Comment (davehensleyns8) [inline assets/configuration/core_configuration.json:18]: Do we want to turn this back on?

Comment (CTDoriott) [inline assets/configuration/core_configuration.json:18]: This should definitely be off. This should be off in production as well: https://github.com/ns8inc/protect-sdk-php/blob/master/assets/configuration/core_configuration.json

Comment (CTDoriott) [reply assets/configuration/core_configuration.json:18]: This should definitely be off. This should be off on master as well: https://github.com/ns8inc/protect-sdk-php/blob/master/assets/configuration/core_configuration.json#L18

File: assets/configuration/core_configuration.json (hunk 1 above)
Comment (crfroehlich) [reply assets/configuration/core_configuration.json:2]: Yes, I think so--but probably not today?

**Pull request merged**